hypervisor/arch/x86/guest/vmcall.c

```diff
@@ -118,7 +137,19 @@ struct acrn_vm *parse_target_vm(struct acrn_vm *service_vm, uint64_t hcall_id, u
 	switch (hcall_id) {
 	case HC_CREATE_VM:
 		if (copy_from_gpa(service_vm, &cv, param1, sizeof(cv)) == 0) {
-			vm_id = get_vmid_by_uuid(&cv.uuid[0]);
+			vm_id = get_vmid_by_name((char *)cv.name);
+			/* if the vm-name is not found, it indicates that it is not in pre-defined vm_list.
```

Comment:
Where is this pre-defined vm_list?

Reply:
it is in this file (vm_configurations.c) that is generated by the config-tool